index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="style.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@300&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Nixie+One&display=swap" rel="stylesheet">
+</head>
+<body>
+	<header>
+		<nav>
+			<button>home.</button>
+			<button>about.</button>
+			<button>projects.</button>
+			<button>contact me.</button>
+		</nav>
+	</header>
+	<main>
+		<div class="top" id="fade">
+		<h1>maria</h1>
+		<h1>gusova.</h1>
```

Comment:
Capital for name and maybe the subheading? other than that the site looks very nice and the coding is well laid out for easy reading.